common/src/main/java/com/wynntils/mc/utils/ComponentUtils.java

```diff
@@ -222,4 +222,8 @@ public static List<Component> splitComponent(Component component, int maxWidth)
         if (split.isEmpty()) split.add(new TextComponent(""));
         return split;
     }
+
+    public static String stripFormattingCodes(String msg) {
```

Comment:
Why is this better than `ChatFormatting#stripFormattingCodes`?

Reply:
Well, it's not, it's actually slightly worse. 😄 I just didn't know about it. I looked through our Utils class but did not think Vanilla would have anything like that...